src/components/Editor/EditorTab.css

```diff
@@ -0,0 +1,17 @@
+.tab {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 50px;
+  cursor: pointer;
+
+  &.active {
+    border-right: 3px solid #029cfd;
```

Comment:
You can use variables so users will be able to customize it

Reply:
hmm it's a good idea to have variables for root of the addon
I'll try to add it as a future task